### PR TITLE
[Moooore IPP] Remove Payments Feature Announcement (Red Badge) and Simple Payments from Orders

### LINKED ISSUE #8182: Remove red badge from Payments icon and Menu tab, and Simple Payments from Orders
### Red Badge
The red dot introduced in the project doesn't bring much at the moment aside from the confusion for the new users. As per the numbers the old users have seen this by now, while the new users open the app and see a red dot right away, which might be confusing.
Because of this, we have to remove the red badge and related functionality code.

### Simple Payments in Orders
We introduced a new navigation back then, in order to teach people that payments have moved
Orders
Plus
Collect payments
Opens the "more" menu tab
Most of our old users have seen this by now, but this is quite confusing for the new users.

### PR DESCRIPTION
**Pull request by toupper**
Closes: #8182 

### Description

With this PR we remove the logic for adding a red badge to announce payments in the Menu tab and Payments Icon. The generic logic of the red badge rendering is left to be used for future feature announcements.
Furthermore, we remove any reference to Simple Payments in the Orders tab:
- Tapping + opens the Orders detail directly without the menu containing Simple Payments.
- Orders banner text is updated to remove any reference to Simple Payments.
#### Test that there is no red badge shown anymore

1. Perform a clean app install.
2. Login
3. There should not be any red badge in the Menu tab
4. Tap on the Menu tab
5. There should not be any red badge in the Payments Icon

#### Test that simple payments is gone from the orders tab

1. Go to the Orders screen
2. Tap on the + button
3. It should open a new order without any reference to simple payments. Finish creating the order and dismiss.
4. The order should appear in the Orders list
5. Check that there are no references to simple payments in the order list top banner with new title: 'Create Orders!'

#### No Simple Payments in Orders

https://user-images.githubusercontent.com/1864060/203379061-48676013-006d-451f-891e-aa717a486398.mp4

---
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift [hunk 1]
```diff
@@ -358,17 +358,25 @@ private extension OrdersRootViewController {
...
             self.dismiss(animated: true) {
                 self.navigateToOrderDetail(order)
             }
         }
-        coordinatingController.start()
+
+        let viewController = OrderFormHostingController(viewModel: viewModel)
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
+            let newOrderNC = WooNavigationController(rootViewController: viewController)
+            navigationController.present(newOrderNC, animated: true)
```
Comment (iamgabrielma) [inline WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift]: Nit: Should we rename this variable for clarity? Something like `newOrderNavigationController` instead?

**Review by iamgabrielma — APPROVED** [review]
Thanks for removing all that code! 🚢 

Nit: There's a reference to the Simple Payments flow we can remove here as well : 

https://github.com/woocommerce/woocommerce-ios/blob/11651935418eafd9a80676639782b592048c7231/WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift#L354

File: WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift (hunk 1 above)
Comment (toupper) [reply WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift]: Great idea! Changed in 3adae6bec14662a7a3a9a21a1f9fc724974c0703

**Comment by toupper** [comment]
@iamgabrielma  Thanks for the review! Indeed, there were still some Simple Payments references in the Orders classes, I removed them in 79c5cdb1315c34c9f53b8c14e58bb5798b4de469. Nice catch!

**Pull request merged**